plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java

```diff
@@ -148,8 +148,10 @@ private void doFinish(String containerName, String fileName, String taskName, St
         IResource[] memberList = container.members();
         for (IResource member : memberList) {
             IFile memberFile = (IFile) member;
-            if (memberFile.getFileExtension().equals("ht")) {
-                throwCoreException("An artifact must contain only one human task file");
+            if (memberFile != null) {
+                if (memberFile.getFileExtension().equals("ht")) {
```

Comment:
Thanks. This was fixed in https://github.com/wso2/devstudio-tooling-bps/pull/30/commits/cef965f069ba0e79f3e0c51787fd38ec6d9d0087